File: Trader/ClientApp/src/models/StockPrice.ts
import { DecimalPipe } from '@angular/common';

export class StockPrice {
    stockSymbol: string;
    CompanyName: string;
    date: Date;
    openMarket: DecimalPipe;
    highMarket: DecimalPipe;
    lowMarket: DecimalPipe;
    closeMarket: DecimalPipe;
    volumeOfMarket: DecimalPipe;
  }

File: VirtualTrading_LaunchPad/VirtualTrading.Model/Stockprice.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace VirtualTrading.Model
{
    public class StockpriceModel
    {
        public string StockSymbol { get; set; }
        public string StockCompanyName { get; set; }
        public DateTime Date { get; set; }
        public decimal? OpenMarket { get; set; }
        public decimal? HighMarket { get; set; }
        public decimal? LowMarket { get; set; }
        public decimal? CloseMarket { get; set; }
        public decimal? VolumeOfMarket { get; set; }
    }
}
